src/components/common/OfficeAddresses.vue

```diff
@@ -43,7 +43,7 @@
                   <br />
                   <v-btn
                     class="reset-btn"
-                    color="red"
+                    color="error"
```

Comment:
"Error" is actually a little darker than red, but it's the app colour for errors, and now this is consistent, too.